.trivyignore

```diff
@@ -18,3 +18,6 @@ CVE-2022-3821
 # ruby vulnerabilities
 CVE-2023-28756
 CVE-2023-28755
+
+# runc vulnerabilities
```

Comment:
this is telegraf vulnerability and move this under telegraf so we can track and handle these through telegraf upgrade,